src/app/header/header.component.css

```diff
@@ -25,6 +25,59 @@
     box-shadow: 0 0 1px 1px #e3e3e3ae;
 }
 
+#offcanvasStartButton {
+    border: 1px solid rgb(202, 202, 202);
+    padding: 3px 5px;
+    border-radius: 6px;
+}
+
 #offcanvasStartButton:hover {
     background-color: #e3e3e3;
 }
+
+hr {
+    background-color: #353434;
+    height: 1px;
+    border: none;
+    margin-bottom: 0;
+    margin-top: 0;
+}
+
+.navText {
+    display: flex;
+    justify-content: start;
+    list-style: none;
+    height: 50px;
+    padding: 1rem;
+    margin-bottom: 0;
+}
+
+a {
```

Comment:
I'd avoid using elements explicitly. From what I notice, all these elements have the `navText` class, right? I'd consider using that instead (e.g. because additional anchors might be added in the future that might not need this styling).